tests/server.test.js

```diff
@@ -2,10 +2,9 @@ const request = require('supertest')
 const server = require('../server')
 
 describe('index route', () => {
-  test('it runs', async done => {
+  it('it runs', async () => {
```

Comment:
For your async events, add `done`. 
Eg. it('it runs', async done => {
    const res = await request(server).get('/')
    expect(res.text).toBe('Api is running!!')
    expect(res.status).toBe(200)
    done()
  })

Reply:
ok